diff -ruN cmake/ectrans_compile_options.cmake cmake/ectrans_compile_options.cmake
--- cmake/ectrans_compile_options.cmake	2024-12-19 15:57:26.000000000 +0000
+++ cmake/ectrans_compile_options.cmake	2025-02-04 15:31:14.075992285 +0000
@@ -25,6 +25,9 @@
   ecbuild_add_fortran_flags("-hnomessage=878")  # A module named ... has already been directly or indirectly use associated into this scope
   ecbuild_add_fortran_flags("-hnomessage=867")  # Module ... has no public objects declared in the module, therefore nothing can be use associated from the module.
   ecbuild_add_fortran_flags("-M7256")           # An OpenMP parallel construct in a target region is limited to a single thread.
+elseif( CMAKE_Fortran_COMPILER_ID MATCHES "IntelLLVM" )
+  ecbuild_add_fortran_flags("-march=core-avx2 -no-fma" BUILD BIT)
+  ecbuild_add_fortran_flags("-fp-model precise -fp-speculation=safe")
 elseif( CMAKE_Fortran_COMPILER_ID MATCHES "Intel" )
   ecbuild_add_fortran_flags("-march=core-avx2 -no-fma" BUILD BIT)
   ecbuild_add_fortran_flags("-fast-transcendentals -fp-model precise -fp-speculation=safe")
